src/pages/[marketplace]/deals.tsx

```diff
@@ -0,0 +1,163 @@
+import { ReactElement, useCallback, useEffect, useState } from "react";
+import { useRouter } from "next/router";
+import { Deal, DealStatus, Ratio, useCredixClient } from "@credix/credix-client";
+import { toUIAmount, formatRatio, formatTimestamp } from "../../utils/format.utils";
+import { Tabs } from "@components/Tabs";
+import { TabPane } from "@components/TabPane";
+import { Table } from "@components/Table";
+import { Icon, IconName } from "@components/Icon";
+import { Slider } from "@components/Slider";
+import { Button } from "@components/Button";
+import Link from "next/link";
+import Big from "big.js";
+import { useLocales } from "../../hooks/useLocales";
+import { useStore } from "@state/useStore";
+import Layout from "@components/Layout";
+import { NextPageWithLayout } from "pages/_app";
+
+const dealsTableColumns = [
+	{
+		title: "Name",
+		icon: "stacked-column-down" as IconName,
```

Comment:
if you use the wrong string here it doesn't complain because you are telling typescript what it is. we should avoid 'as' as much as possible because we lose type safety

Reply:
@BartDeCaluwe should have been request changes instead of approve

Reply:
if you use the wrong string here it doesn't complain because you are telling typescript what it is. we should avoid 'as' as much as possible because we lose type safety